webpack.config.js

```diff
@@ -47,6 +47,13 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
         export: 'default',
       },
     },
+    webexcc: {
+      import: `${path.resolve(__dirname)}/packages/webex/src/contact-center.js`,
+      library: {
+        name: 'Webex',
```

Comment:
Shouldn't this have a different name?

Reply:
We are exporting Webex from the contact-center.js file. Hence, the name has to be Webex